src/policies/conditions.ts

```diff
@@ -152,7 +211,7 @@ class ERC721Ownership extends ContractCondition {
     chain: 'ethereum',
     method: 'ownerOf',
     parameters: [],
-    standardContractType: 'ERC721',
+    standardContractType: 'erc721',
```

Comment:
I guess your linter lowercased this?

Reply:
That was actually changed by hand. I will change it back since we need to maintain compatibility between Python too.